test/data/good_assertion.xml

```diff
@@ -49,9 +48,9 @@
       <AttributeValue>Test Student</AttributeValue>
     </Attribute>
   </AttributeStatement>
-  <AuthnStatement AuthnInstant="2014-03-12T21:35:05.354Z" SessionIndex="_3">
+  <AuthnStatement AuthnInstant="2014-03-12T21:35:05.354Z" SessionIndex="_3" SessionNotOnOrAfter="2014-03-13T22:35:05.387Z">
     <AuthnContext>
       <AuthnContextClassRef>urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport</AuthnContextClassRef>
     </AuthnContext>
   </AuthnStatement>
-</Assertion>
+</Assertion>
```

Comment:
Missing final newline

Reply:
Nick found that these missing newlines are coming from upstream and we haven't touched them.  So let's leave them as-is.